### PR TITLE
Closes #1627 Skip az_paragraphs update if there are no paragraphs.

### LINKED ISSUE #1627: Site with zero paragraphs, cannot complete update for card breakpoints.
## Problem/Motivation
Running database update on sites without any paragraphs results in a neverending message that looks like this.

```
➜  azs-honors git:(master) ✗ lando drush updb -y
 --------------- ---------------------------------------------- --------------- ----------------------------------------------------------------------------- 
  Module          Update ID                                      Type            Description                                                                  
 --------------- ---------------------------------------------- --------------- ----------------------------------------------------------------------------- 
  az_news         920301                                         hook_update_n   920301 - Ensure the better_exposed_filters module is installed.              
  az_paragraphs   920301                                         hook_update_n   920301 - Update card deck breakpoints for tablet and desktop media queries.  
  pathauto        8108                                           hook_update_n   8108 - Update node type conditions from node_type to entity_bundle.          
  text            9201                                           hook_update_n   9201 - Add allowed_formats setting to existing text fields.                  
  user            9301                                           hook_update_n   9301 - Change the users table to use an serial uid field.                    
  block           replace_node_type_condition                    post-update     Updates the node type visibility condition.                                  
  media           modify_base_field_author_override              post-update     Updates stale references to Drupal\media\Entity\Media::getCurrentUserId.     
  node            modify_base_field_author_override              post-update     Updates stale references to Drupal\node\Entity\Node::getCurrentUserId.       
  node            rebuild_node_revision_routes                   post-update     Rebuild the node revision routes.                                            
  system          delete_authorize_settings                      post-update     Remove obsolete system.authorize configuration.                              
  system          sort_all_config                                post-update     Sort all configuration according to its schema.                              
  taxonomy        clear_views_argument_validator_plugins_cache   post-update     Clear the cache after deprecating Term views argument validator.             
  user            update_roles                                   post-update     Calculate role dependencies and remove non-existent permissions.             
  views           sort_identifier                                post-update     Add the identifier option to all sort handler configurations.                
 --------------- ---------------------------------------------- --------------- ----------------------------------------------------------------------------- 


 // Do you wish to run the specified pending updates?: yes.                                                             

>  [notice] Update started: az_news_update_920301
>  [notice] Better Exposed Filters module installed.
>  [notice] Update completed: az_news_update_920301
>  [notice] Update started: az_paragraphs_update_920301
>  [warning] Division by zero az_paragraphs.install:75
>  [notice] Breakpoint settings updated on 0 paragraphs.
>  [warning] Division by zero az_paragraphs.install:75
>  [notice] Breakpoint settings updated on 0 paragraphs.
>  [warning] Division by zero az_paragraphs.install:75
>  [notice] Breakpoint settings updated on 0 paragraphs.
>  [warning] Division by zero az_paragraphs.install:75
>  [notice] Breakpoint settings updated on 0 paragraphs.
>  [warning] Division by zero az_paragraphs.install:75
>  [notice] Breakpoint settings updated on 0 paragraphs.
>  [warning] Division by zero az_paragraphs.install:75
>  [notice] Breakpoint settings updated on 0 paragraphs.
>  [warning] Division by zero az_paragraphs.install:75
>  [notice] Breakpoint settings updated on 0 paragraphs.
>  [warning] Division by zero az_paragraphs.install:75
>  [notice] Breakpoint settings updated on 0 paragraphs.
>  [warning] Division by zero az_paragraphs.install:75
>  [notice] Breakpoint settings updated on 0 paragraphs.
>  [warning] Division by zero az_paragraphs.install:75
>  [notice] Breakpoint settings updated on 0 paragraphs.
>  [warning] Division by zero az_paragraphs.install:75
````
### Describe the bug
This has to do with this line in quickstart.
https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_paragraphs/az_paragraphs.install#L75

### To Reproduce
Run database update for az_paragraphs #920301 on a site with zero paragraphs.

## Proposed resolution
Catch if `$sandbox['current']` equals zero before dividing.

### Expected behavior

If zero paragraphs on site, skip update.

### PR DESCRIPTION
**Pull request by trackleft**
## Description
Return early if no update is needed instead of dividing by zero.

### CODE REVIEW

**Comment by trackleft** [comment]
This has been tested on a site without any paragraphs.

File: modules/custom/az_paragraphs/az_paragraphs.install [hunk 1]
```diff
@@ -28,6 +28,10 @@ function az_paragraphs_update_920301(&$sandbox) {
     ->range($sandbox['current'], $sandbox['current'] + $paragraphs_per_batch)
     ->execute();
 
+  if (empty($paragraph_ids)) {
```
Comment (joeparsons) [inline modules/custom/az_paragraphs/az_paragraphs.install:31]: Maybe we should move this inside the if block that initializes the sandbox (with the count query)?

Comment (joeparsons) [reply modules/custom/az_paragraphs/az_paragraphs.install:31]: (I'm just thinking having it here could cause the message to get logged on sites that do have paragraphs)

Comment (trackleft) [reply modules/custom/az_paragraphs/az_paragraphs.install:31]: Checking

**Review by joeparsons — APPROVED** [review]
Makes sense.  Thanks.

**Comment by trackleft** [comment]
Updated pr works.

**Review by joshuasosa — APPROVED** [review]
I also just ran into this issue today while updating a site with no paragraphs. I applied the change in the latest PR of comparing total to a string and was able to successfully complete the update. Updates also ran as expected on other AZQS-based sites with paragraphs.

**Review by jwmoore1 — APPROVED** [review]

**Pull request merged**